homeassistant/components/group/fan.py

```diff
@@ -110,18 +110,13 @@ def __init__(self, unique_id: str | None, name: str, entities: list[str]) -> Non
         self._percentage = None
         self._oscillating = None
         self._direction = None
-        self._supported_features = 0
+        self._attr_supported_features = 0
```

Comment:
Should this be set as a class instance instead?

Reply:
After checking, it's already the case in FanEntity so removed the default here.